Documentation.md

```diff
@@ -1,78 +1,86 @@
+Those archives are then passed to Chromatic’s normal build process, which screenshots those archives in parallel in whichever [cloud browsers](https://www.chromatic.com/docs/browsers) you like, compares the output, and presents you with the changes to review.
 
-Additionally, the archives you create during each test run can be viewed in a Storybook UI, which means you can inspect them on [chromatic.com](http://chromatic.com) after each build to more closely debug changes and errors.
+Additionally, the archives you create during each test run can be viewed in a Storybook UI, which means you can inspect them on [chromatic.com](http://chromatic.com) (or [locally](#running-the-storybook-locally)) after each build to more closely debug changes and errors.
 
 ## Installation
 
-Get started by installing the archiver package, and the archive storybook:
+Get started by installing the archiver package and the archive Storybook (a specially-configured Storybook instance that can display your archives):
 
 ```bash
 yarn add --dev @chromaui/test-archiver @chromaui/archive-storybook
 ```
 
 ## Configuration
 
-To create archives during playwright tests, import `test` and `expect` from `@chromaui/test-archiver`:
+To create archives during Playwright tests, import `test` and `expect` from `@chromaui/test-archiver` instead of `@playwright/test`:
 
-```tsx
-import { test, expect } from '@chromaui/test-archiver';
+```diff
+- import { test, expect } from '@playwright/test';
++ import { test, expect } from '@chromaui/test-archiver';
 
 // and use as normal
 test('...', async ({ page }) => {
   expect(/* things */);
 });
 ```
 
-Once the above is configured, Chromatic will archive the final state of every test you run irrespective of whether it passes or fails.
+Once the above is configured, Test Archiver will run in addition to your Playwright tests, archiving the final state each one, irrespective of whether it passes or fails.
 
 ### Manual snapshots
 
 To take manual snapshots at specific points of your tests, you can use the `takeArchive` function inside your test runs:
 
-```tsx
+```ts
 import { test, expect, takeArchive } from '@chromaui/test-archiver';
 
+//                               👇 Add testInfo parameter
 test('my test', async ({ page }, testInfo) => {
   await page.goto('https://playwright.dev/');
 
-  // Each time you call takeArchive, we add another archive to your Storybook
+  // Call takeArchive to take an archive "snapshot" of the page at this point
+  //                      👇 Pass testInfo to takeArchive
   await takeArchive(page, testInfo);
 
   await page.getByRole('link', { name: 'Get started' }).click();
 
-  // So you can call it several times, as necessary
-  // To help disambiguate, you can give it a name
+  // You can call it several times, as necessary
+  // To help disambiguate, you can give the archive "snapshot" a name
   await takeArchive(page, 'After clicking link', testInfo);
```

Comment:
_Not sure where else to comment this_

Once you take more than one archive in a test, it makes sense to also be able to name the final, implicit snapshot. Not sure what that API would look like, though...

Reply:
Yeah, agree, not sure what the answer is there really. Maybe we shouldn't take the last snapshot automatically if you take one manually?